bootstrap.php

```diff
@@ -137,6 +140,33 @@ function nfd_create_event_queue_table() {
 	dbDelta( $sql );
 }
 
+/**
+ * Update affiliation code in option table
+ *
+ * @param object $container Container information
+ */
+function nfd_update_options_table( $container ) {
+	$brand_code = array(
+		'bluehost'        => '86241',
+		'hostgator'       => '57686',
+		'web'             => '86239',
+		'crazy-domains'   => '57687',
+		'hostgator-india' => '57686',
+		'bluehost-india'  => '86241',
+		'hostgator-latam' => '57686',
+		'default'         => '86240',
```

Comment:
The `mojo` brand would map to `default`

Reply:
Because we don't have any Jetpack affiliation code information, we're mapping the Mojo brand as default along with other brands.